src/symbol.ts

```diff
@@ -8,16 +10,17 @@ export interface BSONSymbolExtended {
  * @public
  * @category BSONType
  */
-export class BSONSymbol {
-  get _bsontype(): 'Symbol' {
-    return 'Symbol';
+export class BSONSymbol extends BSONValue {
+  get _bsontype(): 'BSONSymbol' {
+    return 'BSONSymbol';
   }
```

Comment:
This is a public API we're changing, let's make sure we mention this in the migration guide.

Reply:
This is tricky, it is not public API it's only function is to power the serializer, but we also cannot mark it as internal because there would be a lot of TS that would complain b/c of it.

But yea I can add it to the migration guide it just is a bit confusing it was never meant to be consumed outside the library / driver

Reply:
> we also cannot mark it as internal

The driver inspects _bsontype at runtime and uses it in the `Filter` type logic